optking/tests/test_opthelper.py

```diff
@@ -26,16 +28,25 @@ def test_step_by_step():
     }
     psi4.set_options(psi4_options)
 
-    opt = optking.OptHelper("hf", comp_type="psi4")
-    opt.build_coordinates()
+    opt = optking.CustomHelper(h2o)
 
     for step in range(30):
-        opt.energy_gradient_hessian()
+
+        grad, wfn = psi4.gradient("hf", return_wfn=True)
+        opt.gX = grad.np.reshape(-1)
+        opt.E = wfn.energy()
```

Comment:
This is better.  Even if messier for user, there was too much going on in that energy_gradient_hessian function.